javascript/code-challenge-13/brackets.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const {Stack} = require('../code-challenge-10/stack-and-queue');
+
+
+function brackets(str) {
+  let allBrackets = new Stack();
+  // let squareBrackets = new Stack();
+  // let curlyBrackets = new Stack();
+
+
+  let inputArr = str.split('');
+  let i = 0;
+  while(inputArr[i]) {
+    if(inputArr[i] === '('){
```

Comment:
This works great! You could probably consolidate all these else statements into a couple of "or" statements. 

if (inputArr[I] === '('   ||   inputArr[I] === '{'   || inputArr[I] === '[' ) {...

Same with checking for the closing brackets, but this is just cleaning up your code and doesn't impact the efficiency!